test/transparentproxy/config.go

```diff
@@ -80,14 +80,14 @@ var defaultTProxyConf = TransparentProxyConfig{
 	IPV6: false,
 }
 
-func InitTproxyConfig() {
-	TProxyConfig = defaultTProxyConf
+func init() {
```

Comment:
Don't use init it's an anti pattern that burnt you once already.

Reply:
addressed